core/model.py

```diff
@@ -1,21 +1,22 @@
-import numpy as np
 import tensorflow as tf
 from . import vggish_input
 from . import vggish_params
 from . import vggish_postprocess
 from . import vggish_slim
-from . import mel_features
+from config import DEFAULT_EMBEDDING_CHECKPOINT, DEFAULT_PCA_PARAMS, MODEL_META_DATA as model_meta
+from maxfw.model import MAXModelWrapper
 
-from config import DEFAULT_EMBEDDING_CHECKPOINT, DEFAULT_PCA_PARAMS
 
-class ModelWrapper(object):
+class ModelWrapper(MAXModelWrapper):
     """
     Contains core functions to generate embeddings and classify them.
-    Also contains any helper function required. 
+    Also contains any helper function required.
     """
 
+    MODEL_META_DATA = model_meta
```

Comment:
I think the metadata should just be defined here instead of the config.

Reply:
Do you think I should add `from config import MODEL_NAME, MODEL_LICENSE` or should I define those variable here?

Reply:
Just define those here. I don't think those variables are used that much anyway.